app/views/layouts/_head.html.haml

```diff
@@ -1,4 +1,6 @@
 %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}
+%head
```

Comment:
why creating another head tag here instead an already head tag? inspect the HTML source code